packages/discussions/src/utils/posts/index.ts

```diff
@@ -51,25 +44,6 @@ export function parseDiscussionURI(discussion: string): IDiscussionParams {
   };
 }
 
-/**
- * Determines if the given user has sufficient privileges to delete the given post
- * @param post An IPost object
- * @param channel An IChannel object
- * @param user An IUser object
- * @returns true if the user can delete the post
- */
-export function canDeletePost(
```

Comment:
need to re-export this at the top of the file? (see `canModifyPost`)

Reply:
done